modules/local/merge_reports/main.nf

```diff
@@ -0,0 +1,32 @@
+process MERGE_REPORTS {
+    tag "$benchmark_tool"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-2076f4a3fb468a04063c9e6b7747a630abb457f6:fccb0c41a243c639e11dd1be7b74f563e624fcca-0':
+        'biocontainers/mulled-v2-2076f4a3fb468a04063c9e6b7747a630abb457f6:fccb0c41a243c639e11dd1be7b74f563e624fcca-0' }"
+
+    input:
+    tuple val(benchmark_tool), path(inputs)
+
+    output:
+    tuple val(benchmark_tool),path("*.summary"), emit: summary
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    merge_reports.py $inputs \\
+        -b $benchmark_tool \\
+        -o ${benchmark_tool}.summary
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        python: \$(python --version | sed 's/Python //g')
+    END_VERSIONS
+    """
+}
```

Comment:
Can you add a stub? This will make it easier to test in the future :)

Reply:
okay stub added :D